app/javascript/views/referrals/referralsInformation/ReferralsInformation.tsx

```diff
@@ -344,6 +348,19 @@ function ReferralsTable(props) {
   const rows = [];
 
   props.referralCodes.forEach((referralCode, index) => {
+    let downloads = 0;
```

Comment:
Could you reuse the function you defined above?
`const { downloads, installs, thirtyDayUse } = processStats(referralCode)`

Reply:
Added to my backlog.